Module not found: Error: Cannot resolve 'file' or 'directory' ../../red

In my Windows 10, i am getting below error.

ERROR in ./build/red/nodes/registry/loader.js
Module not found: Error: Cannot resolve 'file' or 'directory' ../../red in D:\pagenodes\build\red\nodes\registry
 @ ./build/red/nodes/registry/loader.js 30:10-33

31 May 18:23:57 - [error] Error: Cannot find module "../../red"
    at webpackMissingModule (http://localhost:1337/ui.bundle.js:71672:76)
    at Object.init (http://localhost:1337/ui.bundle.js:71672:162)
    at Object.init (http://localhost:1337/ui.bundle.js:59134:13)
    at Object.init (http://localhost:1337/ui.bundle.js:59020:15)
    at http://localhost:1337/ui.bundle.js:56419:19
    at tryCatchReject (http://localhost:1337/ui.bundle.js:58799:31)
    at runContinuation1 (http://localhost:1337/ui.bundle.js:58758:5)
    at Fulfilled.when (http://localhost:1337/ui.bundle.js:58546:5)
    at Thenable.Pending.run (http://localhost:1337/ui.bundle.js:58437:14)
    at Scheduler._drain (http://localhost:1337/ui.bundle.js:58950:20)

--- a/src/red/nodes/registry/loader.js
+++ b/src/red/nodes/registry/loader.js
@@ -52,7 +52,7 @@ events.on("node-locales-dir", function (info) {
 function init(_settings) {
     settings = _settings;
     localfilesystem.init(settings);
-    RED = require('../../red');
+    RED = require('../../pn');
 }
 
 function load(defaultNodesDir, disableNodePathScan) {

